.github/workflows/checkstatus.yml

```diff
@@ -0,0 +1,27 @@
+name: Check all tests
+
+on:
+  status:
+
+jobs:
+  Check-statuses:
+    if: ${{ contains(github.event.branches.*.name, 'master') == 0 }}
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Check status
+      run: |
+        curl -s --url https://api.github.com/repos/${{ github.repository }}/commits/${{ github.event.sha }}/status \
+        --header 'authorization: Bearer ${{ secrets.GITHUB_TOKEN }}' | head -n 2 | tail -n 1 | grep "success"
+    - name: Set up Python
+      if: ${{ success() }}
+      uses: actions/setup-python@v1
```

Comment:
Is this something that github actions provides?  Why `@v1`?  When should `@v1` be changed?

Reply:
it's this one https://github.com/actions/setup-python and there is never version. @acomodi why do we use v1?

Reply:
No reason actually, I will switch to version 2